pkg/util/tls.go

```diff
@@ -0,0 +1,78 @@
+package util
+
+import (
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"net"
+	"net/http"
+
+	pb "github.com/buildbarn/bb-storage/pkg/proto/configuration/server"
+
+	"google.golang.org/grpc/credentials"
+)
+
+// ValidateTls checks params are in order, return non-nil if not.
```

Comment:
As a general (nit) note: could you make sure all comments are full phrases without abbreviations? That will make it clearer imho